beefy-gadget/src/error.rs

```diff
@@ -0,0 +1,9 @@
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
+	/// Check signature error
+	#[error("Message signature {0:?} by {1:?} is invalid.")]
```

Comment:
How long is the `Vec<u8>` here? can we maybe hex-convert before displaying? Otherwise it will be really hard to read.

Reply:
Signature is now hex-converted to a string